apps/SageAccountWeb/.env.development

```diff
@@ -1,5 +1,5 @@
-VITE_SYNAPSE_REPO_ENDPOINT=https://repo-dev.dev.sagebase.org
-VITE_SYNAPSE_PORTAL_ENDPOINT=https://dev.synapse.org
+VITE_SYNAPSE_REPO_ENDPOINT=https://repo-staging.prod.sagebase.org
```

Comment:
will revert before merge

Reply:
You can pass a `--mode=staging` flag to `pnpm start` to tell Vite to use the `.env.staging` file.

Reply:
Reverted (now using flag to point to staging locally)